packages/block/package.json

```diff
@@ -19,7 +19,7 @@
   "scripts": {
     "build": "../../config/cli/ts-build.sh",
     "prepublishOnly": "npm run clean && npm run build && npm run test",
-    "clean": "rm -Rf ./dist && rm -Rf ./dist.browser",
+    "clean": "../../config/cli/clean-package.sh",
```

Comment:
Ah, no, this is actually not serving my needs, I would want to keep this local and not always delete builds from all packages, since then I always have to recreate my local environment after publishing a package.

I would switch this back to an extended local version along my next PR, if you had some strong reasoning for this change in semantics on this script let me otherwise know.

Reply:
Update: ah, sorry, seems that this works actually only locally when invoked from the package folder (just tested)? Ok, then everything is fine! 😄